### DISCUSSION TITLE
On which level to create NWB files

### DISCUSSION THREAD

**Discussion by frederikrogge** [NeurodataWithoutBorders/helpdesk#12] [Q&A] [answered]
Hey, I have a very basic question regarding the NWB files. It says files should be created on an experiment level. In my lab, the calcium imaging recordings per day (one session, same setup, parameters, etc.) can be further split up into different runs which are not continuous in time meaning there are small breaks between them. 

What would be the NWB-way of handling that data? Create a file for each of the runs or one file for an entire day that contains multiple runs? The processing (suite2p) is going to be done on a day-level (or even across days) so it will be the same cells for all runs.

Thank you very much in advance.

**Comment by oruebel** [comment, marked as answer]
One file per day seems appropriate in this case @bendichter @rly

**Reply by frederikrogge** [reply]
Okay, thank you. But how would that be done best with NWB? Having multiple TimeSeries objects for every data type doesn't seem the right thing to do.

**Reply by bendichter** [reply]
@frederikrogge 

If there are a small number of "runs", e.g. <6, it would be fine to separate them out into different TimeSeries objects. You could also concatenate the different files into a single TimeSeries per data source, and use the timestamps of those TimeSeries objects to show discontinuity. Either approach would work.

Both of these approaches would require you to know the start time of the acquisition relative to the session start time (i.e. the start time of the first run).

Either way, you should use the epochs table to indicate the different "runs" (we'd probably call them epochs), and link to the appropriate TimeSeries objects as a column of the epochs table.

**Reply by frederikrogge** [reply]
Thank you for this explanation! I chose to concatenate the data now and use TimeIntervals to save the start and end of each of the runs/epochs.
